src/libs/CurrencyUtils.ts

```diff
@@ -97,11 +97,12 @@ function convertToFrontendAmountAsInteger(amountAsInt: number, currency: string
  *
  * @note we do not support any currencies with more than two decimal places.
  */
-function convertToFrontendAmountAsString(amountAsInt: number | null | undefined, currency: string = CONST.CURRENCY.USD): string {
+function convertToFrontendAmountAsString(amountAsInt: number | null | undefined, currency: string = CONST.CURRENCY.USD, withDecimals = true): string {
+    // function convertToFrontendAmountAsString(amountAsInt: number | null | undefined, withDecimals = true): string {
```

Comment:
@koko57 It seems this is a mistake, right?

Reply:
yeah 🤦🏻‍♀️ fixed!